packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json

```diff
@@ -381,10 +381,12 @@
   "hosting_tab_databases_dumps_error_fetch": "Une erreur est survenue lors de la récupération de vos sauvegardes.",
   "hosting_tab_DATABASES_configuration_copy_title": "Sélectionnez la base de données de destination",
   "hosting_tab_DATABASES_configuration_copy_action_button": "Copier la base de données",
+  "hosting_tab_DATABASES_configuration_information_message": "Que faire si aucune base de données de destination n’apparait ?",
   "hosting_tab_DATABASES_configuration_copy_step1_question_part1": "Votre base de données va être copiée vers une nouvelle destination. Si aucune base de données de destination n'est disponible, vous pouvez en acheter une",
   "hosting_tab_DATABASES_configuration_copy_step1_question_part2": "et l'activer.",
   "hosting_tab_DATABASES_configuration_copy_step1_question_link": "ici",
   "hosting_tab_DATABASES_configuration_copy_step1_select_database": "Copier vers une base de données :",
+  "hosting_tab_DATABASES_configuration_copy_step1_select_hosting_placeholder": "Séléctionner un hébergement",
```

Comment:
```suggestion
  "hosting_tab_DATABASES_configuration_copy_step1_select_hosting_placeholder": "Sélectionner un hébergement",
```

Reply:
Oh dear ! I copy paste again and dit not see that thks!